tests/unit/dataactvalidator/test_a34_appropriations.py

```diff
@@ -0,0 +1,33 @@
+from tests.unit.dataactcore.factories.domain import SF133Factory
+from tests.unit.dataactcore.factories.staging import AppropriationFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+
+_FILE = 'a34_appropriations'
+_TAS = 'a34_appropriations_tas'
+
+
+def test_column_headers(database):
+    expected_subset = {'row_number', 'budget_authority_unobligat_fyb', 'sf_133_amount'}
+    actual = set(query_columns(_FILE, database))
+    assert (actual & expected_subset) == expected_subset
+
+
+def test_success(database):
+    """ Tests that SF-133 amount for line 2490 for the end of the last fiscal year
+    equals Appropriation budget_authority_unobligat_fyb """
+
+    sf = SF133Factory(line=2490, tas=_TAS, period=12, fiscal_year=2015, amount=1)
+    ap = AppropriationFactory(tas=_TAS, budget_authority_unobligat_fyb=1)
```

Comment:
Heh, here's where I'm getting lost, and it might be because I haven't written of the new unit tests yet...

How does the test know how to check the year w/o having a submission object? Thank you for helping a wayward unit test soul understand this magic.